src/webring.rs

```diff
@@ -99,6 +107,25 @@ struct WebringData {
     ordering: Vec<Member>,
```

Comment:
You could probably actually replace the implementation of `WebringData` with an `IndexMap` if you wanted. That could be a future thing though since this PR is already quite big.

Reply:
That's a good idea. It does sound like a big change though so I'll leave that for a separate PR.

Reply:
Completed in deb04e31b3da26633efc2eefda6c69434774a812.